### DISCUSSION TITLE
Can I get the tag infomation that is on the columns via API?

### DISCUSSION THREAD

**Discussion by Makin9** [open-metadata/OpenMetadata#18405] [Q&A] [answered]
I'm trying to get the tables and columns information via API.
They include infomations such as description and tag, etc. that are on the table and columns.
The document states that the API that is called 'Get the latest table profile' is able to get the above information.
I could get all the informations except the tag information that is on the columns.
The document states the API is able to get the tag information but I was not able to do so.
Is this normal?

OpenMetaData version : 14.6

**Comment by Prajwal214** [comment, marked as answer]
@Makin9 You can use Export Table API to fetch the column tags
https://docs.open-metadata.org/swagger.html#operation/exportTable
